straxen/rundb.py

```diff
@@ -266,9 +266,13 @@ def _scan_runs(self, store_fields):
             query = {'number': {'$gt': self.minimum_run_number}}
         else:
             query = {}
+        projection = strax.to_str_tuple(list(store_fields))
+        projection = [f1 for f1 in projection
```

Comment:
Can we change this to: projection = [f1 for f1 in projection
                     if not any([f2.startswith(f1+".") for f2 in projection])]?
That would cover the edge case of two fields that just start with the same characters but are not related.

Reply:
Yeah good catch, let me add it